cmd/test/main.go

```diff
@@ -166,6 +168,16 @@ func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error
 	return invocationFailedErr
 }
 
+func countConfigurations(filesConfigurations []*extractor.FileConfiguration) int {
```

Comment:
I think we should move this into the format function in evaluator if no one else is using filesConfigurations parameter down the line